ssup-evs-ros_wrapper/docker/Dockerfile

```
@@ -0,0 +1,129 @@
+######################################
+
+ARG SENSCORD_PACKAGE_NAME
+ARG CATKIN_WS_DIR="/root/catkin_ws"
+ARG SENSCORD_BASE_DIR=${CATKIN_WS_DIR}/src/evs_ros_wrapper/sensor_sdk/${SENSCORD_PACKAGE_NAME}
+
+
+# Copy all files located under the current directory
+# where the docker build command was executed to the target directory.
+# So, `docker build` command must be executed in the root directory of the project.
+# The distributed Makefile moves the current directory when making.
+ADD . ${CATKIN_WS_DIR}/src/evs_ros_wrapper
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)